src/dotnet/Azure.ClientSdk.Analyzers/Azure.ClientSdk.Analyzers/Descriptors.cs

```diff
@@ -40,5 +40,12 @@ internal class Descriptors
         public static DiagnosticDescriptor AZC0008 = new DiagnosticDescriptor(
             "AZC0008", "ClientOptions should have a nested enum called ServiceVersion",
             "Client type should have a nested enum called ServiceVersion", "Usage", DiagnosticSeverity.Warning, true);
+
+        public static DiagnosticDescriptor AZC0101 = new DiagnosticDescriptor(
```

Comment:
I started non-guideline-specific rules with an offset of 100.

Reply:
That was going to be my first question! :)

Is there any risk that the number of guideline-specific rules will exceed 100?

Reply:
I'll make it 1000.